bin/helpers/capabilityHelper.js

```diff
@@ -241,6 +241,14 @@ const validate = (bsConfig, args) => {
       logger.warn(Constants.validationMessages.SPEC_TIMEOUT_NOT_PASSED_ERROR);
     }
 
+    if(!Utils.isUndefined(bsConfig.run_settings["record"])) {
+      if(Utils.isUndefined(bsConfig.run_settings.projectId)) {
+        logger.warn(Constants.validationMessages.PROJECT_ID_MISSING);
+      } else if (Utils.isUndefined(bsConfig.run_settings["record-key"])) {
```

Comment:
Let's make this a separate `if` condition and not `else if`. The reason for this is to let users know on the first try only they are missing both projectId and record-key if they are missing.